docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,442 @@
+---
+id: molecular-data-prep
+title: Preparing and Validating Molecular Data
+platform_key: DOCS_PREPARING_MOLECULAR_DATA
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The DCC requires that metadata (necessary information describing molecular data files) is submitted at the same time as the molecular data to facilitate automated downstream analysis. This metadata is submitted to the ARGO metadata repository, called Song.  To prepare the molecular metadata, you will need to follow these steps: 
+
+1. Prepare metadata payloads according to the correct schema.
+2. Validate the payloads and molecular files with the ARGO `seq-tools` client. 
```

Comment:
We can also add a github link here for `seq-tools`.

Reply:
we could add it here, or link it to the below section where we show install through git - what do you think is better?

Reply:
We can add a in-page link.